docs/src/euclidean_interface.md

```diff
@@ -71,8 +78,12 @@ Return `v` where $p^v$ is the highest power of $p$ dividing $f$.
 gcd(f::MyElem, g::MyElem)
 ```
 
-Return a greatest common divisor of $f$ and $g$. The return is expected to be
-unit normalized such that if the return is a unit, that unit should be one.
+Return a greatest common divisor of $f$ and $g$.
+
+!!! note
+    For best compatibility with the internal assumptions made by
+    AbstractAlgebra, the return is expected to be unit normalized in such a
```

Comment:
Perhaps we could hyphenate unit-normalised so it doesn't look like a typo for "a unit normalised".

Reply:
sure